packages/ayzek-api-telegram/index.ts

```diff
@@ -20,7 +20,6 @@ const ApiUser = t.interface({
 	id: t.number,
 	username: t.string,
 	first_name: t.union([t.null, t.string]),
-	last_name: t.union([t.null, t.string]),
```

Comment:
Change its type to `t.union([t.undefined, t.string])`

last_name is transferred by api, but if there is no - then this field is omitted, not null

Probally same thing should be done with first_name